java/java.lsp.server/src/org/netbeans/modules/java/lsp/server/protocol/Server.java

```diff
@@ -154,7 +158,18 @@ public static NbLspServer launchServer(Pair<InputStream, OutputStream> io, LspSe
             .setInput(io.first())
             .setOutput(io.second())
             .wrapMessages(processor)
-//                .traceMessages(new java.io.PrintWriter(System.err))
+            .configureGson(gb -> {
```

Comment:
`configureGson` is already called couple of lines above - registrations should be merged.